_data/simple-icons.json

```diff
@@ -1587,6 +1587,11 @@
             "hex": "00665E",
             "source": "https://www.astonmartin.com"
         },
+        {
+            "title": "Astra",
+            "hex": "FFC300",
+            "source": "https://wpastra.com"
+        },
```

Comment:
This should actually probably be sourced from https://wpastra.com/brand-assets/. They have logos there, and the first listed brand color seems more accurate than the yellow: `#5c2ede`.

Reply:
![Screenshot 2024-06-07 at 10 03 56 (Finder)](https://github.com/simple-icons/simple-icons/assets/47499684/d83de064-de1c-4e03-99d8-3db4286d3188)

You sure?

Reply:
Ah yes, I just downloaded the icons zip and there were only svgs. 
![image](https://github.com/simple-icons/simple-icons/assets/19874562/8f5d98a3-4130-445f-9428-6f532e185bd3)


You're right about the logo zip though. There are svg files present. I just updated color and source.

Reply:
Hmm well actually none of the logos here match the outline treatment you've done here. Maybe change the source back if we want to stick with it this way... or we could change to the filled in treatment? Up to you.

Reply:
`astra_logo03.svg` from the logo zip file matches the outline version of the logo.

![image](https://github.com/simple-icons/simple-icons/assets/19874562/1b2cd733-018d-4fc1-9ecb-9eb70d99ce24)

So, I think we can leave the source and icon as they are? What do you think @uncenter?

Reply:
From what I saw the outline was significantly thinner then your outline? I could have been wrong though I didn't fully check.

Reply:
Really? I didn't fully check it either. Anyway, I did a version without the outline. This is how it looks like now.
![image](https://github.com/simple-icons/simple-icons/assets/19874562/460caf15-2088-4a62-83b2-d178bd2630b8)